src/bin/vip-dev-env.js

```diff
@@ -26,4 +26,5 @@ command( {
 	.command( 'list', 'Provides basic info about all local dev environments' )
 	.command( 'exec', 'Execute an operation on a dev environment' )
 	.command( 'import', 'Import data into a local WordPress environment' )
+	.command( 'sync', 'Sync a local development environment with a production environment' )
```

Comment:
```suggestion
	.command( 'sync', 'Pull data from production to local development environment' )
```

Could we align with similar terminology and language as set in the Dashboard?
<img width="779" alt="image" src="https://user-images.githubusercontent.com/14042477/215781594-1e921f2e-1197-463f-a108-f8eedbed2289.png">